modules/identity/pages/single-sign-on-with-oidc.adoc

```diff
@@ -181,6 +181,7 @@ NOTE: Activating this option means any user authorized by the IdP to access Boni
  ** the `ssl-required` property value may need to be changed if Bonita Runtime and the IdP are not both accessed via HTTPS. Possible values for this property are: `all`, `external`, and `none`. For `all`, all requests must come in via HTTPS. For `external`, only non-private IP addresses must come over via HTTPS. For `none`, no requests are required to come over via HTTPS (not recommended for production). For instance, using Azure AD as OIDC provider requires this property to be set to `all`.
  ** the `principal-attribute` value indicates the OIDC Tokens attribute/claim to use to be considered as user identifier in the client application. It should match the username in Bonita. Supported values are: `sub`, `preferred_username`, `email`, `name`, `nickname`, `given_name`, `family_name`. Using another value will result in `sub` claim being used.
  ** `public-client` needs to be set to `false` if the OIDC provider requires the client requests to be authenticated (with a client secret for example). In this case you also need to use the `credentials` property. If this property is set to `true`, then no client Authentication header will be sent to the OIDC provider.
+// TODO all links to keycloak 18 docs --> 404 error
  ** `enable-cors` is required only if you want to be able to use Bonita REST API from a web application located on a different domain from your Bonita installation's. It comes with additional options `cors-max-age`, `cors-allowed-methods`, `cors-allowed-headers`, `cors-exposed-headers`. You can check Keycloak https://www.keycloak.org/docs/18.0/securing_apps/#_java_adapter_config[client adpter config documentation] for more information.
  ** `credentials` defines the https://openid.net/specs/openid-connect-core-1_0.html#ClientAuthentication[authentication method] used for token requests to the OIDC provider when Bonita client is configured with a confidential access type to the OIDC provider (`public-client` is set to `false`). You can set this property with an object containing just a `secret` (in this case `client_secret_basic` authentication method is used) or, since Bonita 2022.1-u6, you can specify the authentication method explicitly (for older versions see https://www.keycloak.org/docs/18.0/securing_apps/#_client_authentication_adapter[Keycloak's client authentication documentation] for the configuration of this property). Supported keys are: `client-secret-basic`, `client-secret-post`, `client-secret-jwt` (or `secret-jwt`), `private-key-jwt` (or `jwt`). Here are some examples:
```

Comment:
**suggestion:** 
```suggestion
// TODO all links to keycloak 18 docs --> 404 error
 ** `credentials` defines the https://openid.net/specs/openid-connect-core-1_0.html#ClientAuthentication[authentication method] used for token requests to the OIDC provider when Bonita client is configured with a confidential access type to the OIDC provider (`public-client` is set to `false`). You can set this property with an object containing just a `secret` (in this case `client_secret_basic` authentication method is used) or, since Bonita 2022.1-u6, you can specify the authentication method explicitly (for older versions see https://www.keycloak.org/docs/18.0/securing_apps/#_client_authentication_adapter[Keycloak's client authentication documentation] for the configuration of this property). Supported keys are: `client-secret-basic`, `client-secret-post`, `client-secret-jwt` (or `secret-jwt`), `private-key-jwt` (or `jwt`). Here are some examples:
```